template/config/file.sls

```diff
@@ -13,12 +13,10 @@ include:
 template-config-file-file-managed:
   file.managed:
     - name: {{ template.config }}
-    - source: {{ files_switch(
-                    salt['config.get'](
-                        tplroot ~ ':tofs:source_files:template-config-file-file-managed',
-                        ['example.tmpl', 'example.tmpl.jinja']
-                    )
-              ) }}
+    - source: {{ files_switch(namespace='template-config-file-file-managed',
+                              source_files=['example.tmpl', 'example.tmpl.jinja']
```

Comment:
Using `namespace=` and `source_files=` seems to make sense at this stage.  However, I think it just makes this longer than in needs to be.  Not pushing this issue, just opening it for discussion.

Reply:
Maybe `lookup=` is much more meanful than `namespace=`.